src/applications/mhv/medications/components/shared/StatusDropdown.jsx

```diff
@@ -6,13 +6,13 @@ const StatusDropdown = props => {
 
   const content = () => {
     switch (status) {
-      case 'active': {
+      case 'Active': {
         return (
```

Comment:
I would keep using `refillStatus` here. So that we use `refillStatus` for validation and `dispStatus` for visual display.